stdlib/math.pyi

```diff
@@ -112,7 +112,10 @@ def log1p(__x: _SupportsFloatOrIndex) -> float: ...
 def log2(__x: _SupportsFloatOrIndex) -> float: ...
 def modf(__x: _SupportsFloatOrIndex) -> tuple[float, float]: ...
 
-if sys.version_info >= (3, 9):
+if sys.version_info >= (3, 12):
+    def nextafter(__x: _SupportsFloatOrIndex, __y: _SupportsFloatOrIndex, *, steps_after: int | None = None) -> float: ...
```

Comment:
- The new keyword argument is `steps`, not `steps_after`
- It takes anything that has an `__index__` method, not just `int`

```suggestion
    def nextafter(__x: _SupportsFloatOrIndex, __y: _SupportsFloatOrIndex, *, steps: SupportsIndex | None = None) -> float: ...
```

Reply:
Oops, not sure what happened there, I've been testing most of these specifically too. Thanks for catching!